test/helpers/db.js

```diff
@@ -9,18 +9,10 @@ const Notification = require('../../src/models/db/notification').Notification
 const Fulfillment = require('../../src/models/db/conditionFulfillment').ConditionFulfillment
 
 exports.init = function * () {
+  yield knex.migrate.rollback(knexConfig).then()
```

Comment:
Do you have steps to reproduce? I don't get index errors when I rollback before running migrations, and it works on circleci.